include/openPMD/Mesh.hpp

```diff
@@ -174,6 +174,10 @@ class Mesh : public BaseRecord< MeshRecordComponent >
     template< typename T >
     Mesh& setTimeOffset(T timeOffset);
 
+    /** Access to records.
+     */
+    Container< MeshRecordComponent > records;
```

Comment:
Please revert the addition in this file since it's a bit outdated.
Records and record components are accessed like this instead:
https://github.com/openPMD/openPMD-api/blob/0.2.0-alpha/examples/2_read_serial.py#L34-L44